### ISSUE TITLE
Manually set global last-changed tick

### ISSUE THREAD

**Issue by UkoeHB** [lifescapegame/bevy_replicon#109]
There is a category of games that have limited conditions where the server world will be modified. For those games, it is feasible to manually track the last tick where something in the world was modified (or an event sent to clients). Once the last world-change-tick has been acked, it is a waste of CPU time to scan the ECS world for individual changes.

It would be nice if we could manually define/inject the last world-change-tick to the server replication loop, either on a global or per-client basis. We can then short-circuit replication if a client has acked that tick. Moreover, even if a client has not acked a tick, if we have cached replication buffers for the tick range (last acked, last world-change], then we don't need to scan the world (just send the buffers again).

**Comment by Shatur** [comment]
So it's basically pause replication after specific tick? Make sense to me.
I think it should be done globally and once we will have rooms - per room.

**Comment by Shatur** [comment]
No longer relevant after #16.

**Comment by Shatur** [comment]
But there is not such thing as last changed tick. I fill like it's something that should be solved on Bevy side. They are considering adding change detection on archetype and entity levels.
